src/main/java/Main.java

```diff
@@ -294,6 +299,19 @@ private static void _main(String[] args) throws Exception {
         mainMethod.invoke(null,new Object[]{arguments.toArray(new String[0])});
     }
 
+    /**
+     * reads up to maxRead bytes from InputStream if available into a String
+     *
+     * @param in
```

Comment:
All empty Javadoc tags need to be either documented or removed.
Otherwise it will cause Javadoc warnings